modules/directives/src/main/scala/scala/build/preprocessing/directives/Toolkit.scala

```diff
@@ -50,9 +55,11 @@ object Toolkit {
       s.isEmpty || s.contains(Constants.toolkitOrganization) || s.contains(scala)
   }
 
-  case class ToolkitDefaults(
+  case class ToolkitDefinitions(
     isScalaToolkitDefault: Boolean = false,
-    isTypelevelToolkitDefault: Boolean = false
+    scalaToolkitExplicitVersion: Option[String] = None,
+    isTypelevelToolkitDefault: Boolean = false,
+    typelevelToolkitExplicitVersion: Option[String] = None
```

Comment:
Isn't this field redundant?

Reply:
it sort of is, but I feel it'd be confusing if it wasn't in place... I'd rather leave it.
But yeah, it's technically unused right now.